src/shared-components/forms/style.js

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+import { COLORS } from '../../constants';
+
+export const FormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 16px;
+`;
+
+export const errorIconStyles = css`
+  position: relative;
+
+  svg {
+    display: block;
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    top: 50%;
+    margin-top: -8px;
+    right: 16px;
+    fill: ${COLORS.statusRed};
+  }
+`;
+
+export const InputContainer = styled.div`
+  svg {
+    display: none;
+  }
+
+  ${({ shouldRenderError }) => shouldRenderError && errorIconStyles};
+`;
+
+export const baseInputStyles = css`
+  appearance: none;
+  border: 1px solid ${COLORS.border};
+  border-radius: 0;
+  color: ${COLORS.primary};
```

Comment:
body text should always be `purple85`

Reply:
Ok. Note that I refactored this Form component to a more clean simpler to use version

Reply:
I put primaryTint1 which in turn will be purple85 once colors branch is merged